2021-04-25 18:05:09,178 INFO    MainThread:3832 [wandb_setup.py:_flush():69] setting env: {}
2021-04-25 18:05:09,178 INFO    MainThread:3832 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-25 18:05:09,179 INFO    MainThread:3832 [wandb_init.py:_log_setup():336] Logging user logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_180509-l1lkj0nd/logs/debug.log
2021-04-25 18:05:09,179 INFO    MainThread:3832 [wandb_init.py:_log_setup():337] Logging internal logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_180509-l1lkj0nd/logs/debug-internal.log
2021-04-25 18:05:09,179 INFO    MainThread:3832 [wandb_init.py:init():369] calling init triggers
2021-04-25 18:05:09,179 INFO    MainThread:3832 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-25 18:05:09,180 INFO    MainThread:3832 [wandb_init.py:init():418] starting backend
2021-04-25 18:05:09,180 INFO    MainThread:3832 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-25 18:05:09,196 INFO    MainThread:3832 [backend.py:ensure_launched():123] starting backend process...
2021-04-25 18:05:09,202 INFO    MainThread:3832 [backend.py:ensure_launched():128] started backend process with pid: 3845
2021-04-25 18:05:09,204 INFO    MainThread:3832 [wandb_init.py:init():423] backend started and connected
2021-04-25 18:05:09,206 INFO    MainThread:3832 [wandb_init.py:init():463] updated telemetry
2021-04-25 18:05:09,207 INFO    MainThread:3832 [wandb_init.py:init():482] communicating current version
2021-04-25 18:05:09,915 INFO    MainThread:3832 [wandb_init.py:init():487] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-25 18:05:09,915 INFO    MainThread:3832 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-25 18:05:10,103 INFO    MainThread:3832 [wandb_init.py:init():520] starting run threads in backend
2021-04-25 18:05:10,183 INFO    MainThread:3832 [wandb_run.py:_console_start():1524] atexit reg
2021-04-25 18:05:10,184 INFO    MainThread:3832 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-25 18:05:10,185 INFO    MainThread:3832 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-25 18:05:10,187 INFO    MainThread:3832 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-25 18:05:10,188 INFO    MainThread:3832 [wandb_init.py:init():544] run started, returning control to user process
2021-04-25 18:05:35,524 INFO    MainThread:3832 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-25 18:05:35,525 INFO    MainThread:3832 [wandb_run.py:_restore():1466] restore
2021-04-25 18:05:37,710 INFO    MainThread:3832 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 18:05:38,088 INFO    MainThread:3832 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 18:05:38,388 INFO    MainThread:3832 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2030
  total_bytes: 2030
}

2021-04-25 18:05:38,490 INFO    MainThread:3832 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2030
  total_bytes: 2030
}

2021-04-25 18:05:38,593 INFO    MainThread:3832 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2030
  total_bytes: 2030
}

2021-04-25 18:05:38,695 INFO    MainThread:3832 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2030
  total_bytes: 2030
}

2021-04-25 18:05:38,798 INFO    MainThread:3832 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2030
  total_bytes: 2030
}

2021-04-25 18:05:38,900 INFO    MainThread:3832 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2030
  total_bytes: 2030
}

2021-04-25 18:05:40,018 INFO    MainThread:3832 [wandb_run.py:_show_files():1838] logging synced files
